core/src/main/java/hudson/model/RootAction.java

```diff
@@ -50,4 +50,17 @@ public interface RootAction extends Action, ExtensionPoint {
     default @CheckForNull Badge getBadge() {
         return null;
     }
+
+    /**
+     * Identifies if the action as a primary action.
+     * Primary actions may be handled differently in the UI (for existence by always showing on the header rather than in an actions dropdown).
```

Comment:
```suggestion
     * Primary actions may be handled differently in the UI (for example by always showing on the header rather than in an actions dropdown).
```